setup.py

```diff
@@ -29,5 +29,6 @@ def readme():
         url = "http://archman.github.io/beamline/",
         scripts  = [os.path.join('scripts',sn) for sn in scriptnames],
         requires = requiredpackages,
```

Comment:
Then this line should be removed.

Reply:
Ok, I did not know the keyword hence I left it in. I will change that.